brackend/endpoints/tournaments/tournaments.py

```diff
@@ -0,0 +1,36 @@
+from flask import Blueprint, request, jsonify
+
+from flask_restful import Api
+from flask_restful import Resource
+
+from brackend.tasks.tasks import (
+    save_new_tournament,
+    get_tournament_by_id,
+)
+
+tournament_bp = Blueprint('tournaments', __name__)
+tournament_api = Api(tournament_bp)
+
+
+class Tournaments(Resource):
+    """
+        Create a new tournament
+    """
+    def post(self):
+        body = request.get_json()
+        name = body.get("name")
+        new_tourny = save_new_tournament(name)
+        return jsonify(new_tourny)
```

Comment:
I think this is a good example of where a task doesn't really make sense here, we want the process to be:

1. User sends request to make a new tournament
2. We handle request, create the tourny
3. We only send back a successful response after we know the tournament has definitely been created
4. Response has the new tournament info/id, so we can update the mobile app's state with the new tourny

Reply:
makes sense, thanks